mkdocs.yml

```diff
@@ -93,7 +93,7 @@ nav:
       - Coding Guide: developer-guide/developer.md
       - OVN-Kubernetes Container Images: developer-guide/image-build.md
       - Documentation Guide: developer-guide/documentation.md
-      - Local Testing Guide: developer-guide/testing.md
+      - Local Testing Guide: installation/kind.md
```

Comment:
so this is not the right place I'd say..

1. remove the file getting-started/launching-ovn-kubernetes.md
2. Move/rename your current file to installation/launching-ovn-kubernetes-on-kind.md
3. Update the point https://ovn-kubernetes.io/governance/CONTRIBUTING/#development-environment-setup to reflect the new path
4. Launching OVN-Kubernetes: installation/launching-ovn-kubernetes.md that's the place we want the website to point to

The Local Testing Guide was meant for a different thing: show a sample example of "how to do a PR change" -> test that on KIND (by linking to the KIND install docs from this testing guide); then checking logs on kind to see the changes done on the PR; then also adding the details around checklist https://ovn-kubernetes.io/governance/CONTRIBUTING/#pull-request-checklist and https://ovn-kubernetes.io/ci/ci/#run-tests ===> this might need more work so might be best as another PR

Reply:
Fixed Suggestions!